humanguide/design-concepts/design-philosophy.rst

```diff
@@ -0,0 +1,55 @@
+=================
+Design philosophy
+=================
+
+The Ubuntu interface has been designed according to a philosophy called Suru.
+
+.. figure:: /_static/images/humanguide/750w_DesignPhilosophy_MainImage.png
+
+
+Suru meaning
+------------
+
+Here at Ubuntu we believe that everyone should have access to free, reliable, and trusted software that can be shared and developed by anyone. This has paved the way to allow the community to grow and prosper with freedom trust and collaboration. This integral belief has been translates into our design philosophy called Suru.
+
+- Suru stems from the Ubuntu brand values alluding to Japanese culture.
+
+- The design of Suru is inspired by origami, because it give us a solid and tangible foundation.

```

Comment:
There's a lot of us/our which I think would have to be reformulated to show the historic origin from some other party (Canonical) that is not us/our

Reply:
Hi! Thanks for your reply. Perhaps, the text sholud be corrected or rewritten. I agree that all Canonical stuff should be reformulated. I don't like that there is a link in Docementation to this chapter, and it leads to page existing only at Webarchive. So I converted it into rst text. Certainly, I can rewrite the whole chapter.

Reply:
Oh I wasn't aware we have such a link. Shouldn't that be changed with this PR as well then?

Reply:
But, I would really like to hear from someone like @cibersheep first, whether this is indeed still our philosophy. Before you spend too much time editing... Hopefully there *is* someone who can answer that :D